apps/dg/views/component_view.js

```diff
@@ -187,14 +202,14 @@ DG.DragBorderView = SC.View.extend(
           return DG.ComponentView.findComponentViewParent(this);
         },
         getContainerWidth: function () {
-          return window.innerWidth; // go global
+          return containerWidth = $('#codap').width();
```

Comment:
`containerWidth` causes a lint error. I think it could be `return $('#codap').width();`

Reply:
Just pushed a commit to fix this.